source/examples/rapids-sagemaker-hpo/notebook.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>I think we had avoided running the cat because it would bake a potentially old container image into the notebook output. We might want to clear this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>Just putting into writing here what we'd talked about earlier today... I think it's alright to keep this cell output. That image tag is already in a bunch of other places throughout the notebook.</p>